### PR TITLE
Updating Browser Checklist

### LINKED ISSUE #31: Browser testing checklist
Let's add a browser testing checklist with as a resource.

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
If any CFPBers are interested in the numbers, take a look at GHE/flapjack/Getting-started-with-Flapjack/issues/43

## Changes
- Removes IE7 
- Adding Android 4.0

## Review 
- @cfarm 
- @ascott1 

## Notes
Closes #31

### CODE REVIEW

File: browser-checklist.md [hunk 1]
```diff
@@ -2,15 +2,14 @@
...
 - [ ] IE11
 - [ ] IE10
 - [ ] IE9
 - [ ] IE8
-- [ ] IE7
 - [ ] Opera
-- [ ] iOS8
-- [ ] Android 5
+- [ ] iOS 8
+- [ ] Android 4.0
```
Comment (ascott1) [inline browser-checklist.md]: We may want to make this Android 4.0+, since I assume we're also testing later versions of the Android browser.

**Comment by ascott1** [comment]
Looks great @KimberlyMunoz! A small comment above, but much improved. Thanks for digging in to the research!

File: browser-checklist.md (hunk 1 above)
Comment (KimberlyMunoz) [inline browser-checklist.md]: Thanks! Fixed in 9114e84

Comment (ascott1) [inline browser-checklist.md]: :boom:

**Comment by cfarm** [comment]
yesssss

**Pull request merged**